client/src/components/Map.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+import { API_KEY } from '../apis';
+
+const Marker = () => (
+  <i className="pi pi-map-marker" style={{ fontSize: '3em', color: '#fc3903' }}></i>
+);
+
+const defaultProps = {
+  center: {
+    lat: 40.78,
+    lng: -73.97,
+  },
+  zoom: 12,
+};
+
+export function Map(props) {
+  return (
+    <div style={{ height: '70vh', width: '100%' }}>
+      <GoogleMapReact
+        bootstrapURLKeys={{ key: API_KEY }}
```

Comment:
Either we don't need the script in the index.html or we don't need this one.

Reply:
Ah right, ok I changed it!